### PR TITLE
chore(web): add missing Japanese translations

### PR DESCRIPTION
**Pull request by nourbalaha**
# Overview
The missing Japanese translations have been added in this PR

### CODE REVIEW

File: web/src/i18n/translations/ja.yml [hunk 1]
```diff
@@ -182,14 +182,14 @@ Are you sure you want to delete the model: 本当にこのモデルを削除し
...
-Permanently removes your project and all of its contents from Re:Earth CMS. This action is not reversible, so please continue with caution.: ''
-Write something here to describe this record.: このレコードの説明を入力してください
-Project Settings: ''
-Need request: ''
-If this option is chosen, all new model within the project will default follow it: ''
+Delete Project: プロジェクトを削除
+Permanently removes your project and all of its contents from Re:Earth CMS. This action is not reversible, so please continue with caution.: プロジェクトとその内容をRe:Earth CMSから完全に削除します。このアクションは元に戻せません。
+Write something here to describe this record.: このレコードの説明を入力してください。
+Project Settings: プロジェクト設定
+Need request: リクエストの必要性
```
Comment (HideBa) [inline web/src/i18n/translations/ja.yml]:
```suggestion
Need request: リクエストの要否
```

Comment (rot1024) [inline web/src/i18n/translations/ja.yml:189]:
```suggestion
Need request: リクエストが必須
```

File: web/src/i18n/translations/ja.yml [hunk 2]
```diff
@@ -182,14 +182,14 @@ Are you sure you want to delete the model: 本当にこのモデルを削除し
...
+If this option is chosen, all new model within the project will default follow it: このオプションが選択された場合、プロジェクト内のすべての新しいモデルはデフォルトでそれに従います
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:190]:
```suggestion
If this option is chosen, all new model within the project will default follow it: このオプションが選択された場合、プロジェクト内のすべての新しいモデルはデフォルトでそれに従います。
```

File: web/src/i18n/translations/ja.yml [hunk 3]
```diff
@@ -229,7 +229,7 @@ Multi line text: 複数行テキスト
 Markdown text: マークダウン
 Rich text which supports md style: マークダウン対応のリッチテキスト
 Asset file: アセット
-true/false field: ''
+true/false field: true/false フィールド
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:232]:
```suggestion
true/false field: 真偽
```

Note: In Notion, "checkbox" is used.

File: web/src/i18n/translations/ja.yml [hunk 4]
```diff
@@ -289,10 +289,10 @@ Successfully updated request!: リクエストの更新に成功しました。
 Failed to delete one or more items.: アイテムの削除に失敗しました。
 One or more items were successfully deleted!: アイテムの削除に成功しました。
 Successfully updated Request!: リクエストの更新に成功しました。
-Failed to publish items.: ''
-Successfully published items!: ''
-Failed to unpublish items.: ''
-Successfully unpublished items!: ''
+Failed to publish items.: アイテムの公開に失敗しました。
+Successfully published items!: アイテムの公開に成功しました!
+Failed to unpublish items.: アイテムの公開に失敗しました。
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:294]:
```suggestion
Failed to unpublish items.: エラーによりアイテムを非公開にできませんでした。
```

File: web/src/i18n/translations/ja.yml [hunk 5]
```diff
@@ -289,10 +289,10 @@ Successfully updated request!: リクエストの更新に成功しました。
...
+Successfully unpublished items!: アイテムの公開解除に成功しました!
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:295]:
```suggestion
Successfully unpublished items!: アイテムを非公開にしました。
```

File: web/src/i18n/translations/ja.yml [hunk 6]
```diff
@@ -311,8 +311,8 @@ Failed to create field.: フィールドの作成に失敗しました。
 Successfully created field!: フィールドの作成に成功しました。
 Failed to update project.: プロジェクトの更新に失敗しました。
 Successfully updated project!: プロジェクトの更新に成功しました。
-Failed to update request roles.: ''
-Successfully updated request roles!: ''
+Failed to update request roles.: リクエストロールの更新に失敗しました。
+Successfully updated request roles!: リクエストロールの更新に成功！
```
Comment (rot1024) [inline web/src/i18n/translations/ja.yml:315]:
```suggestion
Successfully updated request roles!: リクエストロールの更新に成功しました。
```

File: web/src/i18n/translations/ja.yml (hunk 1 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:189]: Done!

File: web/src/i18n/translations/ja.yml (hunk 2 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:190]: Done!

File: web/src/i18n/translations/ja.yml (hunk 3 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:232]: Done!

File: web/src/i18n/translations/ja.yml (hunk 4 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:294]: Done!

File: web/src/i18n/translations/ja.yml (hunk 5 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:295]: Done!

File: web/src/i18n/translations/ja.yml (hunk 6 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:315]: Done!

**Review by rot1024 — APPROVED** [review]

**Review by issmail-basel — APPROVED** [review]

**Pull request merged**